api/http/handler/helm/helm_repo_search_test.go

```diff
@@ -15,27 +15,42 @@ import (
 )
 
 func Test_helmRepoSearch(t *testing.T) {
+	extended := helper.ExtendedTests(t)
 	is := assert.New(t)
 
 	helmPackageManager := test.NewMockHelmBinaryPackageManager("")
 	h := NewTemplateHandler(helper.NewTestRequestBouncer(), helmPackageManager)
 
 	assert.NotNil(t, h, "Handler should not fail")
 
-	repos := []string{"https://charts.bitnami.com/bitnami", "https://portainer.github.io/k8s"}
+	type testCase struct {
+		repo    string
+		enabled bool
+	}
+
+	tests := []testCase{
+		{"https://charts.bitnami.com/bitnami", false},
+		{"https://portainer.github.io/k8s", true},
+	}
+
+	for _, test := range tests {
+		func(tc testCase) {
+			if tc.enabled || extended {
```

Comment:
should not use this value in the test; better to return early - as it makes code easier to read/follow.

Reply:
I've now removed this functionality.  Skipping the whole test as it is an integration test